frontend/manage_users.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * @file This file contains the Manage function. This function is used for
+ * granting and revoking permissions from users, as well as approving requests
+ * for adding individuals to herds in the genebanks you manage.
+ */
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import { Tabs, Tab } from '@material-ui/core/'
+import { Box, Button } from '@material-ui/core'
+
+import { useDataContext } from './data_context'
+import { ManageUser } from '~manage_user';
+
+// Define styles for tab menu
+const useStyles = makeStyles({
+  sidebar: {
+    float: "left",
+    height: "calc(100% - 50px)",
+    width: "250px",
+    margin: "0",
+    borderRight: `1px solid rgba(0,0,0,0.2)`,
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center"
+  },
+  verticalTabs: {
+    height: "calc(100% - 50px)",
+    width: "250px",
+  },
+  controls: {
+    height: "calc(100% - 50px)",
+    width: "calc(100% - 250px)",
+    padding: "0.5cm 1cm",
+    overflowY: "scroll",
+  },
+  centerButton: {
+    width: "80%",
+  }
+});
+
+export function ManageUsers() {
+  const classes = useStyles()
+  const {users} = useDataContext()
+  const [userTab, setUserTab] = React.useState(0);
+  const [selectedUser, selectUser] = React.useState(undefined as number | string | undefined)
```

Comment:
This should probably not be called `selectedUser` but something with edit action.

You can narrow down the string type to `'new'`: `number | 'new' | undefined`